Client/src/components/contanier/staff/Courses/AddCourse/index.css

```diff
@@ -0,0 +1,63 @@
+.add-course-title {
+  color: #06468f;
+  letter-spacing: 1px;
+  font-size: 34px;
+  font-weight: bold;
+  margin-top: 50px;
+  margin-left: 39px;
+  margin-bottom: 9px;
+}
+.add-course-line {
+  width: 204px;
+  border-bottom: 1.5px solid #06468f;
+  margin-top: 100px;
```

Comment:
@naremanhilles If you'd like, you could collapse these three margin styles into a single line: `margin: 100px 0 73px 39px`.

Reply:
fexed it